src/battleground/HealthBarsField.js

```diff
@@ -0,0 +1,32 @@
+//@flow strict
+
+import * as React from 'react';
+import "../Main.css";
+import HealthBar from './HealthBar.js';
+import Tank from '../tanks/Tank.js';
+
+type Props = {|
+	playerOneTank: Tank,
```

Comment:
The tanks aren't optional here, but they are in HealthBar.js. Remove the question mark in HealthBar.js